common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -839,12 +839,26 @@ div.video {
       background-size: 15%;
     }
 
-    .btn-play {
-      text-indent: -999px;
-      overflow: hidden;
+    .btn-play.btn-pre-roll {
```

Comment:
So we don't mess with tests or other functionality, I've added a class to the button in this case and am providing style overrides here. This isolates this case for future work too.